chapters/03-package-setup.Rmd

```diff
@@ -1,37 +1,55 @@
-# Setting up an R package
+# Setting up an R package {#package-setup}
 
 - Describing what a package is and when or why to make one
+    - Use [available](https://github.com/r-lib/available) to make a name (naming is hard).
 - Using devtools, usethis for development
-- e.g. `use_devtools()` and `use_usethis()`.
+    - e.g. `use_devtools()` and `use_usethis()`.
 - Developing functions
 - Basic explanation of what a function is and its components
 - Actual process of taking code and converting it into a function will
     be done in next chapter
```

Comment:
I don't understand how this section fits into this chapter. We're assuming readers already know what a function is and how they work, and in terms of actually writing functions, there aren't any exercises about it, and we create functions in the next chapter.